homeassistant/components/backup/manager.py

```diff
@@ -207,7 +212,9 @@ async def async_create_backup(
         """
 
     @abc.abstractmethod
-    async def async_get_backups(self, **kwargs: Any) -> dict[str, _BackupT]:
+    async def async_get_backups(
+        self, **kwargs: Any
+    ) -> tuple[dict[str, Backup], dict[str, Exception]]:
```

Comment:
I'd return a dataclass instead of a tuple. That's more robust for changes.

Reply:
I don't think that's motivated when we only ever call this from the WS API